app/styles/app.css

```diff
@@ -176,6 +176,24 @@ code[data-diff] .diff-insertion::before {
   padding: 0.75em;
 }
 
+/* Pullquote (blockquote) emphasis */
```

Comment:
To keep the scope of your work narrow and side effect-free, let's not make style changes in this PR.